openfe/protocols/openmm_utils/omm_settings.py

```diff
@@ -244,13 +255,8 @@ class Config:
 
     timestep: FloatQuantity['femtosecond'] = 4 * unit.femtosecond
     """Size of the simulation timestep. Default 4 * unit.femtosecond."""
-    collision_rate: FloatQuantity['1/picosecond'] = 1.0 / unit.picosecond
+    langevin_collision_rate: FloatQuantity['1/picosecond'] = 1.0 / unit.picosecond
     """Collision frequency. Default 1.0 / unit.pisecond."""
-    n_steps = 250 * unit.timestep  # todo: IntQuantity
-    """
-    Number of integration timesteps between each time the MCMC move
-    is applied. Default 250 * unit.timestep.
-    """
     reassign_velocities = False
```

Comment:
Potentially move to `AlchemicalSamplerSettings` or change doc string to explain what `move` means here.

Reply:
is this for `reassign_velocities`?  seems like an integrator thing

Reply:
reassign velocities is specific to the behaviour that exists when doing multi state moves, not the timestep integration itself, that makes it a property of the sampler not the integrator